Dockerfile

```diff
@@ -7,9 +7,9 @@ COPY poetry.lock /todo-app
 COPY pyproject.toml /todo-app
 
 FROM base as production
-RUN poetry config virtualenvs.create false && poetry install --no-dev
+RUN poetry config virtualenvs.create false --local && poetry install --no-dev
 COPY . /todo-app
-CMD poetry run gunicorn --bind=0.0.0.0:5000 "app:create_app()"
+CMD poetry run gunicorn --bind=0.0.0.0:$PORT "app:create_app()"

```

Comment:
This means the production image no longer works locally without setting $PORT. Either change this to have a default value of 5000 instead of being required, or update the README to mention $PORT.